operator/v1/types_ingress.go

```diff
@@ -545,6 +545,19 @@ type AWSLoadBalancerParameters struct {
 	//
 	// +optional
 	NetworkLoadBalancerParameters *AWSNetworkLoadBalancerParameters `json:"networkLoadBalancer,omitempty"`
+
+	// subnets specifies the list of subnets for the load balancer to
+	// route traffic to. The values can be either a subnet ID or name.
+	// In order for the load balancer to be provisioned with subnets:
+	// * Each subnet must exist.
+	// * Each subnet must be from a different availability zone.
+	// * The load balancer service must be deleted.
+	//
+	// When omitted, the subnets will be auto-discovered per availability zone.
+	//
+	// +optional
+	// +openshift:enable:FeatureGate=IngressControllerLBSubnetsAWS
+	Subnets []AWSSubnetReference `json:"subnets,omitempty"`
```

Comment:
Needs a `listType` tag, its this atomic?

Reply:
I've read the docs (https://github.com/kubernetes/kube-openapi/blob/master/pkg/idl/doc.go and https://kubernetes.io/docs/reference/using-api/server-side-apply/), but I'm struggling to totally grasp the difference between an atomic and set listType. I don't totally understand the implications as it relates to my field. I see set:
```
// Sets are lists that must not have multiple times the same value. Each
// value must be a scalar (or another atomic type).
```

And atomic:
```
// Atomic lists will be entirely replaced when updated. This tag may be
// used on any type of list (struct, scalar, ...).
```

Sounds like atomic only allows one "manager" for server side apply. That seems acceptable for this field.

I also see atomic is the default, so I'll go with that.